src/assets/css/inputs.css

```diff
@@ -659,35 +659,43 @@ input:checked + .input-slider:before {
     cursor: pointer;
     min-width: 200px;
     padding: .125rem .5rem;
+
+    &&:focus {
```

Comment:
Small nitpick: Does `&&` do anything different than `&`?

Could find any reference in https://sass-lang.com/documentation/file.SASS_REFERENCE.html#parent-selector

Reply:
Oh, I don't know to be honest. I've been using it for years and seems to be working. But you're right, it's not really mentioned anywhere.

Just for the record, we're using postcss and whatever it supports.